nevergrad/optimization/base.py

```diff
@@ -488,7 +488,8 @@ def recommend(self) -> p.Parameter:
         """
         recom_data = self._internal_provide_recommendation()  # pylint: disable=assignment-from-none
         if recom_data is None:
-            return self.current_bests["pessimistic"].parameter
+            name = "minimum" if self.parametrization.descriptors.deterministic_function else "pessimistic"
+            return self.current_bests[name].parameter
```

Comment:
@teytaud this is a drastic change in my opinion, you may want to crosscheck it in the xps

Reply:
I think it is safe but yes I'll check xps... complicated though are the absolute indicators are just new.

Reply:
I guess we can wait a bit to merge if it is safer

Reply:
or I provide a fix for the noisy variants